packages/design-system/src/components/datalist/list/list.tsx

```diff
@@ -118,46 +119,50 @@ const OptionList = forwardRef(function OptionList(
    * LAZY OPTIONS LOADING
    */
   // Add option when observed entry is seen
-  const observer = useMemo(
-    () =>
-      new window.IntersectionObserver(
-        (entries) => {
-          if (onObserve) {
-            const newlySeenOptions = entries
-              .filter((entry) => entry.isIntersecting)
-              .map((entry) => entry.target.dataset.option);
-            userSeenOptions.current = addUniqueEntries(
-              userSeenOptions.current,
-              ...newlySeenOptions
-            );
-            onObserve(userSeenOptions.current);
-          }
-        },
-        {
-          root: listRef.current,
-          // Gets ~2 extra items below scroll container BoundingBox
-          rootMargin: '60px',
+  const observer = useMemo(() => {
+    if (!listRef) {
```

Comment:
Should this be

```suggestion
    if (!listRef.current) {
```

?

Reply:
Yes, it should be. Fixing.